drizzlepac/haputils/analyze.py

```diff
@@ -533,6 +539,10 @@ def analyze_data(input_file_list, log_level=logutil.logging.DEBUG, type=""):
             # GYROS mode so some drift)
             generate_msg(input_file, msg_type, no_proc_key, no_proc_value)
         
+        elif science_array_of_zeros:
+            do_process=False
+            log.warning(f'Science data for {input_file} filled with zeros, not aligning.')
```

Comment:
I would change the message slightly to be consistent with the current code.  log.warning(f'Science data for {input_file} filled with zeros.  Dataset cannot be aligned.')